src/iperf_tcp.c

```diff
@@ -44,6 +42,12 @@
 #include "net.h"
 #include "cjson.h"
 
+#if defined(HAVE_INTTYPES_H)
+# include <inttypes.h>
```

Comment:
Nice to know there's a real solution to this problem!  I need to go read about the usage of `<inttypes.h>`.  And thanks for providing a fallback for non-C99-compliant people.

Reply:
Well, in 2017 if people don't have C99 compliance (which was discussed in draft form for 2+ years before approval), then they're almost 20 years out-of-date...  Hopefully this is adequate.